--- entities-list.js
+++ entities-list.js
@@ -11,7 +11,7 @@ class EntitiesList
     static getAll()
     {
         return [
-            'rooms', 'objects', 'skills', 'items', 'ads', 'audio', 'chat',
+            'objects', 'skills', 'items', 'ads', 'chat',
             'config', 'features', 'respawn', 'rewards', 'snippets',
             'teams', 'users'
         ];

--- crud-test-data.js
+++ crud-test-data.js
@@ -37,7 +37,6 @@ class CrudTestData
             audio: {
                 audio_key: testPrefix+'-audio-'+timestamp,
                 room_id: 4,
-                files_name: 'test.mp3',
                 category_id: 1,
                 enabled: 1
             },
